assets/scss/_theme.scss

```diff
@@ -137,3 +137,47 @@ body {
 .modal-open {
   overflow-y: auto !important;
 }
+
+// Restore radio button group styling to allow for buttons no longer being direct children of button groups - which is
+// required for tooltips to display
+.btn-group > {
+  .radio-btn-container {
+    flex-grow: 1;
+    .btn {
+      width: 100%;
+    }
+    &:not(:last-child) > {
+      .btn {
+        border-bottom-right-radius: 0 !important;
+        border-top-right-radius: 0 !important;
+        border-right-width: 0 !important;
+      }
+    }
+    &:not(:first-child) > {
+      .btn {
+        border-bottom-left-radius: 0 !important;
+        border-top-left-radius: 0 !important;
+      }
+    }
+  }
+}
+
+.btn-group-lg > {
+  .radio-btn-container > {
+    .btn {
+      --cui-btn-padding-y: .5rem;
+      --cui-btn-padding-x: 1rem;
+      --cui-btn-font-size: 1.25rem;
+      --cui-btn-border-radius: var(--cui-border-radius-lg);
+    }
+  }
+}
+
+// In vue-select parameter dropdowns, avoid highlighting both focused (used when navigating with keyboard) and
+// hovered menu items, by forcing unhighlighted background colour on all menu items if they have a sibling which
+// is hovered, unless they are the unhovered item is currently selected (also highlighted, in a different colour).
```

Comment:
Some of words may been or perhaps added

Reply:
?? Is this a request to make this comment more comprehensible? It's a fair request!

Reply:
I couldn't grammar this
> unless they are the unhovered item is currently selected